int8 unfinished_task
---
int8 finish_task
